scripts/install_xcpng.py

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/env python3
+import argparse
+import atexit
+import os
+import requests
+import sys
+import tempfile
+import time
+try:
+    sys.path.append(f"{os.path.abspath(os.path.dirname(__file__))}/..")
+except Exception:
+    pass
+from lib.commands import ssh, scp, SSHCommandFailed
+from lib.common import wait_for, is_uuid
+from lib.host import host_data
+from lib.pool import Pool
+from lib.vm import VM
+from packaging import version
```

Comment:
I'd put this global import before the local ones and before the sys thingie.

Reply:
i'm talking about the `packaging` import.
Which is not in our sources.

Reply:
if it's not in our sources, why is it used in differents files ? like conftest.py, host.py, test_cert_inheritance.py.

Reply:
I think you don't understand the problem. We group the imports from installed packaged (global) before relative imports like `from lib.host import ...`.

Reply:
It's just a coding style issue here.